package/yast2-devtools.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu May 14 08:35:30 UTC 2015 - jreidinger@suse.com
+
+- revert incompatible change for vardir as it break yast2.rpm
+  build
```

Comment:
same as @besser82 changes, so if it use old version nothing needed to tested and only if change with ID appear, it deserve submission to SP1